cadc-access-control-server/src/main/java/ca/nrc/cadc/ac/server/web/users/UserActionFactory.java

```diff
@@ -108,7 +108,8 @@ public AbstractUserAction createAction(HttpServletRequest request)
 
                 if (segments.length == 0)
                 {
-                    action = new GetUserListAction();
+                    // s2484: GetUserListAction disabled to align with GDPR
```

Comment:
Just a minor point:  we shouldn't be referencing internal story/task numbers in our open source code because they don't mean anything to our collaborators.

Reply:
ok. I removed the comment. Thanks!